packages/legend-query-builder/src/components/QueryBuilderResultPanel.tsx

```diff
@@ -481,8 +481,19 @@ export const QueryBuilderResultPanel = observer(
       });
     };
     const queryValidationIssues = queryBuilderState.validationIssues;
+    const queryWindowValidationIssues =
+      queryBuilderState.fetchStructureState.implementation instanceof
+      QueryBuilderTDSState
+        ? queryBuilderState.fetchStructureState.implementation.windowState
+            .validationIssues
+        : undefined;
+
+    const queryWindowStateIsValid = !queryWindowValidationIssues;
+
     const isQueryValid =
-      !queryBuilderState.isQuerySupported || !queryValidationIssues;
+      (!queryBuilderState.isQuerySupported || !queryValidationIssues) &&
+      queryWindowStateIsValid;
```

Comment:
dont think this boolean expression is correct :)

Reply:
const  isSupportedQueryValid = queryBuilderState.validationIssues && queryWindowStateIsValid;
const isQueryValid = (!queryBuilderState.isQuerySupported || !isSupportedQueryValid);